models/linkages/man/split.inputs.LINKAGES.Rd

```diff
@@ -4,7 +4,7 @@
 \alias{split.inputs.LINKAGES}
 \title{split.inputs.LINKAGES}
 \usage{
-split.inputs.LINKAGES(settings, start.time, stop.time)
+\method{split}{inputs.LINKAGES}(settings, start.time, stop.time)
```

Comment:
@mdietze @araiho can you take a look?  All I did was re-run the roxygen/make document commands.....

Same here.  I am not sure what was intended?